src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,59 @@
 package core.basesyntax;
 
-class Pair {
+import java.util.Objects;
 
+class Pair<T, V> {
+    private T firstPair;
+    private V secondPair;
```

Comment:
```suggestion
    private T first;
    private V second;
```
they are not pairs, they are fields of Pair class